config/generateRunFile.coffee

```diff
@@ -311,6 +311,14 @@ generateDev = (KONFIG, options) ->
       set +o errexit
     }
 
+    function runMongoDocker() {
+        docker run -d -p 27017:27017 --name=mongo koding/mongo:2016-10-11
```

Comment:
I will update the docker image name after merging this to master - merge will cause a Docker build. see here https://hub.docker.com/r/koding/mongo-auto/~/settings/automated-builds/